todo/src/components/Todo.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 const Todo = (props) => {
   return (
-    <li>
+    <li onDoubleClick={() => props.removecompleted(props.todo)} onClick={() => props.completed(props.todo)} className={ props.todo.completed ? 'line-through' : null}>
```

Comment:
Awesome double click implementation! Very cool to have a couple different ways to interact with the todo items. 😄